contrib/registry/nacos/registry.go

```diff
@@ -158,12 +164,16 @@ func (r *Registry) GetService(ctx context.Context, serviceName string) ([]*regis
 	}
 	items := make([]*registry.ServiceInstance, 0, len(res))
 	for _, in := range res {
+		kind := in.Metadata["kind"]
```

Comment:
always check okness when fetching values from map:
```
if kind, ok := in.Metadata["kind"]; ok {
    ...
}
```

Also, set allowedKind as const values, we dont need nacos to support kinds with its own capability

Reply:
done!